src/cr/cube/cube.py

```diff
@@ -673,7 +679,22 @@ def raw_cube_array(self):
         )
         # ---must be read-only to avoid hard-to-find bugs---
         raw_cube_array.flags.writeable = False
-        return raw_cube_array
+        return raw_cube_array.T if self._require_array_transposition else raw_cube_array
```

Comment:
This is probably going to get more complicated than a simple `x.T` (when we have more dimensions in the future). So maybe add a `"TODO:"` here... or `"NOTE:"`.

Reply:
Also, what happens with other things (not just numbers), like labels and valid indexes? Is it enough to "just" transpose the `raw_cube_array`?

Reply:
that's a valid question and i think this can be checked in the exporter :D